fairseq/modules/multihead_attention.py

```diff
@@ -146,23 +155,19 @@ def forward(
             saved_state = None
 
         if self.self_attention:
-            q = self.q_proj(query)
```

Comment:
For my understanding, was this for performance improvements? If so did it help?

Reply:
yes, this was causing a 10% regression.